editor/store.js

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
Thanks for splitting off this file. I've been wanting to separate reducers from store initialization for a while 👍